src/Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp

```diff
@@ -188,6 +189,9 @@ struct CharacteristicEvolution {
           Actions::UpdateGauge<
               Metavariables::uses_partially_flat_cartesian_coordinates>>,
       Actions::PrecomputeGlobalCceDependencies,
+      tmpl::conditional_t<
```

Comment:
I don't understand why the check is based on partially flat coordinates on not on whether or not you want CCM vs. CCE

Reply:
I added this long and awkward bool template parameter before CCM was ready. Now we might want to rename it to `evolve_ccm` or something like that. Please let me know whether you prefer me to add one more commit to address this, or if you would like me to open a separate PR.